network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/we_found_section.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+<section class="tw-dark tw-my-7 large:tw-my-[120px]">
+  <div class="container tw-max-w-3xl tw-mx-auto">
+    <h4 class="tw-text-lg medium:tw-text-[1.375rem] tw-uppercase tw-font-sans tw-font-bold tw-mb-5">{% trans 'Ultimately, we found:' %}</h4>
+
+    <div class="tw-flex tw-flex-col tw-space-y-5">
+      <div class="tw-flex tw-flex-col medium:tw-flex-row tw-gap-3">
+        <span class="tw-text-4xl tw-text-white">{% trans '01' %}</span>
+        <p class="tw-font-changa tw-text-3xl medium:tw-text-[2.375rem] tw-mb-0">{% blocktrans trimmed %}People overwhelmingly feel that using YouTube’s user controls <span class="tw-text-[#FF3D3D]">does not change</span> their recommendations at all.{% endblocktrans %}</p>
```

Comment:
**Suggestion** Would `strong` instead of `span` be more semantic here, since the emphasis seems to be the intent of the styling?

Reply:
This actually doesn't require bold so i'll leave as is for now haha

Reply:
Just the font makes it look that way visually but i think its okay as a `<p>`

Reply:
Yea true. I was thinking more semantic emphasis for screen readers etc. I guess it would have to be `<strong class="font-normal">` 😅